src/types/profile.js

```diff
@@ -90,6 +90,7 @@ export type SamplesTable = {
   responsiveness: Array<?Milliseconds>,
   stack: Array<IndexIntoStackTable | null>,
   time: Milliseconds[],
+  duration: Milliseconds[],
```

Comment:
My backthinking is that in the future we could want to process durations from the actual "time" values we get. For example we could get a sample earlier than expected, in that case the previous sample would last less than `interval`. On the contrary a sample could happen later than expected, and in that case the previous sample would last more than `interval`. We would accept a tolerance of for example 2 * interval, or 1.5 * interval, and get then more precise values.

OR as you say we could just forget about actual durations and just discuss about counts. :)

In your mind, is `weight` a multiplier then ? The count would be `weight * interval` ?

Reply:
I made the property optional, but kept `duration`. Also because it's optional now and compare profiles aren't shared, I believe we can still switch to a weight later without any migration needed.